bin/api-docs/gen-components-docs/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+const docgen = require( 'react-docgen-typescript' );
+const glob = require( 'glob' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+/**
+ * Internal dependencies
+ */
+const { generateMarkdownDocs } = require( './markdown' );
+
+// For consistency, options should generally match the options used in Storybook.
```

Comment:
Is there a way to read those same options from the storybook config, instead of duplicating them?

Reply:
Unfortunately they're internal. I just pulled them from the source code.